tests/core/pyspec/eth2spec/test/helpers/block_processing.py

```diff
@@ -0,0 +1,60 @@
+def for_ops(state, operations, fn) -> None:
+    for operation in operations:
+        fn(state, operation)
+
+
+def get_process_calls(spec):
```

Comment:
Funny.. this might be useful across many contexts of other tests. Obviously important for the `latest_block_header`  to be correct for sync committee tests, but maybe would have more "correct" states in other context if we use it across the board